modules/cnf-cpu-infra-container.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// scalability_and_performance/cnf-performance-addon-operator-for-low-latency-nodes.adoc
+
+[id="cnf-cpu-infra-container_{context}"]
+= Restricting CPUs for infra and application containers
+
+You can reserve cores, or threads, for infra containers, which perform operating system housekeeping tasks, from a single NUMA node and put your application containers, which run workloads, on another NUMA node. The reason for this is that the housekeeping processes might be using the CPUs in a way that would impact latency sensitive processes running on those same CPUs. Keeping your workloads on a separate NUMA node prevents the processes from interfering with each other. Additionally, each NUMA node has its own memory bus that is not shared.
+
+By default, CRI-O uses all online CPUs to run infra containers on nodes, which can result in context switches and spikes in latency. You can ensure that infra containers and application containers each run on specific CPUs specifying two groups of CPUs in the `spec` section of the performance profile. 
+
+* `isolated` - The application container CPUS for workloads. Have the lowest latency. Processes in this group have no interruptions and so can, for example, reach much higher DPDK zero packet loss bandwidth.
+
+* `reserved` - The infra container CPUs for housekeeping. Threads in the reserved group tend to be very busy, so latency-sensitive applications should be run in the isolated group. See link:https://kubernetes.io/docs/tasks/configure-pod-container/quality-service-pod/#create-a-pod-that-gets-assigned-a-qos-class-of-guaranteed[Create a pod that gets assigned a QoS class of `Guaranteed`].
+
+
+.Procedure
+
+. Create a performance profile that is appropriate for your hardware and topology.
+
+. Add the `reserved` and `isolated` parameters with the CPUs you want reserved and isolated for the infra and application containers:
++
+[source,yaml]
+----
+﻿apiVersion: performance.openshift.io/v2
+kind: PerformanceProfile
+metadata:
+  name: infra-cpus
+spec:
+  cpu:
+    reserved: "0-4,9" <1>
+    isolated: "5-8" <2>
+  nodeSelector: <3>
+    node-role.kubernetes.io/worker: ""
+----
+<1> Specify which CPUs are for infra containers to perform cluster and operating system housekeeping duties.
```

Comment:
nit: I'd drop "are for infra containers" and switch with "which CPUs are for performing cluster..."